apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -803,18 +844,28 @@ export class MarkdownDocumenter {
       }
     }
 
+    if (showInheritedMembers && maybeIncompleteMembers) {
+      output.appendNode(
+        new DocParagraph({ configuration }, [
+          new DocEmphasisSpan({ configuration, italic: true }, [
+            new DocPlainText({ configuration, text: '(The tables below may be missing inherited members)' })
```

Comment:
```suggestion
            new DocPlainText({ configuration, text: '(Some inherited members may not be shown because they are not represented in the documentation.)' })
```

I struggled to come up with a sentence that:
- doesn't create an intriguing mystery about how the members could be "missing"
- doesn't imply that the website maintainers did anything wrong
- looks okay on [a page without any tables](https://github.com/zelliott/rushstack/blob/33d6df2bf0272043a0b14f2a39271ac45c7c9992/build-tests/api-documenter-scenarios/etc/inheritedMembers/markdown/api-documenter-scenarios.extendsclassfromanotherpackage.md)
- leaves open the possibility that we actually did find all the members

Let me know if you have anything better. 😄

Reply:
Done.